File: Fibonacci (Without Recursive)/Python/fibonacci-i.py
x = int(input('Masukkan panjang deret: '))

fib = [0, 1]

for i in range(2, x):
  f1 = fib[i - 2]
  f2 = fib[i - 1]
  deretN = f1 + f2

  fib.append(deretN)
  
print(fib)

File: Fibonacci (Without Recursive)/C/fibonacci-i.c
#include <stdio.h>  
int main (){
  
    int n1 = 0, n2 = 1, n3, panjDeret;
    
    printf("Masukkan Panjang Deret: ");
    scanf("%d", &panjDeret);
    
    printf("%d %d ", n1, n2);
    
    for(int i = 2; i < panjDeret; i++){
        n3 = n1 + n2;
        printf("%d ", n3);
        n1 = n2;
        n2 = n3;
    }
  
    return 0;
}
